istioctl/cmd/proxyconfig.go

```diff
@@ -872,7 +875,7 @@ func bootstrapConfigCmd() *cobra.Command {
 		},
 	}
 
-	bootstrapConfigCmd.Flags().StringVarP(&outputFormat, "output", "o", jsonOutput, "Output format: one of json|yaml")
+	bootstrapConfigCmd.Flags().StringVarP(&outputFormat, "output", "o", jsonOutput, "Output format: one of json|yaml|short")
```

Comment:
Can you please also add `Example` [here](https://github.com/istio/istio/blob/d9ab7bb10dbacdc057b49321bf5b88b2128620e8/istioctl/cmd/proxyconfig.go#L834) on how to use this command. This will be helpful for docs purposes.